doc/releases/changelog-0.11.0.md

```diff
@@ -202,79 +266,157 @@
   for the `StatePrep`, `BasisState`, and `QubitUnitary` operations. For the parameter-shift method,
   this is allowable for all operations.
 
-* Changes to support a dynamic number of qubits:
-
-  - The `qalloc_p` custom JAX primitive can now take in a dynamic number of qubits as a tracer
-    and lower it to mlir.
-    [(#1549)](https://github.com/PennyLaneAI/catalyst/pull/1549)
-
-  - `ComputationalBasisOp` can now take in a quantum register in mlir, instead of an explicit, fixed-size list of qubits.
-    [(#1553)](https://github.com/PennyLaneAI/catalyst/pull/1553)
+* An `mlir_opt` property has been added to `qjit` to access the optimized MLIR representation of a compiled 
+  function.
+  [(#1579)](https://github.com/PennyLaneAI/catalyst/pull/1579)
+  [(#1637)](https://github.com/PennyLaneAI/catalyst/pull/1637)
 
-  - Non-observable measurements without explicit wires will now compile to `ComputationalBasisOp` with a quantum register, instead of the explicit list of all qubits on the device.
-  This means the same compiled IR can be reused even if the device changes its number of qubits across runs.
-  This includes `probs(), state(), sample(), counts()`.
-    [(#1565)](https://github.com/PennyLaneAI/catalyst/pull/1565)
+  ```python
+  from catalyst import qjit
 
-  - In mlir, `ProbsOp, StateOp, SampleOp, CountsOp` ops now carry an optional new SSA operand for their return shapes.
-    This operand is used during bufferization pass to allocate result memrefs dynamically.
-    A new verification is added to check that this new operand and static return shapes cannot coexist.
-    [(#1574)](https://github.com/PennyLaneAI/catalyst/pull/1574)
+  @qjit
+  def f(x):
+      return x**2
+  ```
 
-* Added `qjit.mlir_opt` property to access the optimized MLIR representation of a compiled function.
-  [(#1579)](https://github.com/PennyLaneAI/catalyst/pull/1579)
-  [(#1637)](https://github.com/PennyLaneAI/catalyst/pull/1637)
+  ```pycon
+  >>> f(2)
```

Comment:
This is a _very_ long output!

Reply:
Yes I think it would be sufficient to just print the first few lines maybe?